metrics/xtreme_s/xtreme_s.py

```diff
@@ -0,0 +1,253 @@
+    >>> print({k: round(v, 2) for k, v in results.items()})
+    {'wer': 0.56}
+
+    >>> xtreme_s_metric = datasets.load_metric('xtreme_s', 'covost2')
+    >>> references = ["bonjour paris", "il est necessaire de faire du sport de temps en temp"]
+    >>> predictions = ["bonjour paris", "il est important de faire du sport souvent"]
+    >>> results = xtreme_s_metric.compute(predictions=predictions, references=references)
+    >>> print({k: round(v, 2) for k, v in results.items()})
+    {'bleu': 19.38}
+
+    >>> xtreme_s_metric = datasets.load_metric('xtreme_s', 'fleurs')
+    >>> references = [0, 1, 0, 0, 1]
+    >>> predictions = [0, 1, 1, 0, 0]
+    >>> results = xtreme_s_metric.compute(predictions=predictions, references=references)
+    >>> print({k: round(v, 2) for k, v in results.items()})
+    {'accuracy': 0.6'}
+
+    >>> xtreme_s_metric = datasets.load_metric('xtreme_s', 'minds14')
+    >>> references = [0, 1, 0, 0, 1]
+    >>> predictions = [0, 1, 1, 0, 0]
+    >>> results = xtreme_s_metric.compute(predictions=predictions, references=references)
+    >>> print({k: round(v, 2) for k, v in results.items()})
+    {'f1': 0.5}
+"""
+
+#    >>> print({"pearson": round(results["pearson"], 2), "spearmanr": round(results["spearmanr"], 2)})
+
+_CONFIG_NAMES = ["fleurs-asr", "mls", "voxpopuli", "babel", "covost2", "fleurs-land_id", "minds14"]
+
+try:
+    from jiwer import transforms as tr
+
+    _jiwer_available = True
+except ImportError:
+    _jiwer_available = False
+
+if _jiwer_available and version.parse(importlib_metadata.version("jiwer")) < version.parse("2.3.0"):
+
+    class SentencesToListOfCharacters(tr.AbstractTransform):
+        def __init__(self, sentence_delimiter: str = " "):
+            self.sentence_delimiter = sentence_delimiter
+
+        def process_string(self, s: str):
+            return list(s)
+
+        def process_list(self, inp: List[str]):
+            chars = []
+            for sent_idx, sentence in enumerate(inp):
+                chars.extend(self.process_string(sentence))
+                if self.sentence_delimiter is not None and self.sentence_delimiter != "" and sent_idx < len(inp) - 1:
+                    chars.append(self.sentence_delimiter)
+            return chars
+
+    cer_transform = tr.Compose(
+        [tr.RemoveMultipleSpaces(), tr.Strip(), SentencesToListOfCharacters(SENTENCE_DELIMITER)]
+    )
+elif _jiwer_available:
+    cer_transform = tr.Compose(
+        [
+            tr.RemoveMultipleSpaces(),
+            tr.Strip(),
+            tr.ReduceToSingleSentence(SENTENCE_DELIMITER),
+            tr.ReduceToListOfListOfChars(),
+        ]
+    )
+else:
+    cer_transform = None
+
+
+def simple_accuracy(preds, labels):
+    return float((preds == labels).mean())
+
+
+def f1_and_simple_accuracy(preds, labels):
+    return {"f1": float(f1_score(y_true=labels, y_pred=preds)), "accuracy": simple_accuracy(preds, labels)}
```

Comment:
```suggestion
    return {"f1": float(f1_score(y_true=labels, y_pred=preds, average="macro")), "accuracy": simple_accuracy(preds, labels)}
```
Necessary for multi-class F1